types/office-js-preview/index.d.ts

```diff
@@ -980,15 +980,25 @@ declare namespace Office {
     }
 
     /**
-     * Provides information about what Requirement Sets are supported in current environment.
+     * Provides information about which Requirement Sets are supported in the current environment.
      */
     interface RequirementSetSupport {
         /**
         * Check if the specified requirement set is supported by the host Office application.
         * @param name - Set name; e.g., "MatrixBindings".
-        * @param minVersion - The minimum required version; e.g., "1.4". Note: String type is recommended data type for this parameter. The use of number type is deprecated and will not be compatible with recent requirement sets.
+        * @param minVersion - The minimum required version (e.g., "1.4").
         */
-       isSetSupported(name: string, minVersion?: string | number): boolean;
+       isSetSupported(name: string, minVersion?: string): boolean;
+
+       
+        /**
+        * Check if the specified requirement set is supported by the host Office application.
+        * @deprecated
+        * @param name - Set name; e.g., "MatrixBindings".
+        * @param minVersionNumber - The minimum required version (e.g., 1.4). Warning: Your add-ins should use the string overload of `isSetSupported` instead. 
+        * The use of the number type is deprecated. This is because the JavaScript parser cannot differentiate between numeric values such as 1.1 and 1.10, where as it can for string values such as "1.1" and "1.10".
```

Comment:
I think this is missing something. Maybe it should say "stand for" or "represent"?

```suggestion
        * The use of the number type is deprecated. This is because the JavaScript parser cannot differentiate between numeric values such as 1.1 and 1.10, where as it can stand for string values such as "1.1" and "1.10".
```

Reply:
I'm trying to not be repetitive in this sentence. "The JavaScript parser **cannot** differentiate between numeric values [...], where as is **can** for strings." I think the second "differentiate" is heavily enough implied to avoid repeating it.

Reply:
Ah, then in that case, it should say "whereas"

```suggestion
        * The use of the number type is deprecated. This is because the JavaScript parser cannot differentiate between numeric values such as 1.1 and 1.10, whereas it can for string values such as "1.1" and "1.10".
```